### PR TITLE
feature/CP-6870-ios-app-banners-targeting-by-events-from-previous-sessions

### PR DESCRIPTION
**Pull request by cpandya25**

### CODE REVIEW

**Review by cpKevinrodriguesnobre — APPROVED** [review]

**Review by mmiyakheel — APPROVED** [review]

File: CleverPush/Source/CleverPushSQLiteManager.m [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                return YES;
+            }
+        }
+        sqlite3_close(database);
+    }
+    return NO;
+}
+
+#pragma mark - to get all the recrods from database
+- (NSArray<CPAppBannerEventFilters *> *)getcleverPushDatabaseAllRecords {
```
Comment (gbhrdt) [inline CleverPush/Source/CleverPushSQLiteManager.m:177]: Do we need `cleverPush` in the function names? Cant we do it without cleverPush before or does it has a purpose?
And IF we need it, we should stay camelCase (getCleverPush.... not getcleverPush)

**Review by gbhrdt — CHANGES_REQUESTED** [review]

File: CleverPush/Source/CleverPushSQLiteManager.m (hunk 1 above)
Comment (cpandya25) [reply CleverPush/Source/CleverPushSQLiteManager.m]: I renamed all the classes related to the database and its functions.
There is no need to keep the clever push word before the functions.

**Comment by gbhrdt** [comment]
resolve conflicts @cpandya25

File: CleverPush/Source/CPSQLiteManager.m [hunk 2]
```diff
@@ -0,0 +1,245 @@
+#import "CPSQLiteManager.h"
+#import "CleverPush.h"
+#import "CPLog.h"
+
+@implementation CPSQLiteManager
+
+static CPSQLiteManager *sharedInstance = nil;
+NSString *databaseTable = @"cleverpush_table_banner_track_event";
```
Comment (gbhrdt) [inline CleverPush/Source/CPSQLiteManager.m:8]: Please use same table name + same column names like in Android PR

(I think table e.g. is called cleverpush_tracked_events)

**Review by gbhrdt — CHANGES_REQUESTED** [review]

File: CleverPush/Source/CPSQLiteManager.m (hunk 2 above)
Comment (cpKevinrodriguesnobre) [reply CleverPush/Source/CPSQLiteManager.m:8]: @cpandya25

**Pull request merged**